conf/application.conf

```diff
@@ -40,11 +40,11 @@ play.filters.enabled += play.filters.csp.CSPFilter
 
 play.filters.csp {
   directives {
-    script-src = "'self' 'unsafe-inline' localhost:9032 localhost:12345 www.googletagmanager.com www.google-analytics.com"
-    style-src = "'self' 'unsafe-inline' localhost:9032"
-    img-src = "'self' localhost:9032 region1.google-analytics.com region1.analytics.google.com"
-    font-src = "'self' localhost:9725 localhost:9032"
-    frame-src = "'self'"
+    script-src = "'self' 'unsafe-inline' localhost:9032 localhost:12345 www.google-analytics.com www.googletagmanager.com tagmanager.google.com *.digital.nuance.com"
+    style-src = "'self' 'unsafe-inline' localhost:9032 tagmanager.google.com fonts.googleapis.com *.digital.nuance.com"
+    img-src = "'self' localhost:9032 region1.google-analytics.com region1.analytics.google.com https://ssl.gstatic.com www.gstatic.com www.google-analytics.com *.digital.nuance.com data:"
+    font-src = "'self' localhost:9725 localhost:9032 https://ssl.gstatic.com www.gstatic.com https://fonts.gstatic.com https://fonts.googleapis.com *.digital.nuance.com data:"
+    frame-src = "'self'https://www.googletagmanager.com *.digital.nuance.com"
     connect-src = "'self' region1.google-analytics.com region1.analytics.google.com"
```

Comment:
I think we may need https://www.google-analytics.com in connect-src as well. Potentially looks like current issue in QA is being caused by connect-src blocking something